TTS/_version.py

```diff
@@ -1 +1 @@
-__version__ = "0.0.14"
+__version__ = "0.0.14.1"
```

Comment:
also pls don't change the model version. It'll be updated with the next model release.

Reply:
@erogol Thanks. Changed the version. I think you need these changes from the main branch and their version is 0.0.14.1 in main. 
```
        check_argument("pad", c, prerequest="characters", restricted=True)
        check_argument("eos", c, prerequest="characters", restricted=True)
        check_argument("bos", c, prerequest="characters", restricted=True)
        check_argument("characters", c, prerequest="characters", restricted=True)
```